src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue27200.cs

```diff
@@ -0,0 +1,24 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue27200 : _IssuesUITest
+	{
+		public override string Issue => "The size of the CollectionView header is incorrect when it contains a Binding on an IsVisible";
+
+		public Issue27200(TestDevice device)
+		: base(device)
+		{ }
+
+		[Test]
+		[Category(UITestCategories.CollectionView)]
+		public void CollectionViewHeaderSizewithIsVisibleBinding()
+		{
+			App.WaitForElement("collectionView");
+
+			VerifyScreenshot();
```

Comment:
The test is failing on Windows:
![image](https://github.com/user-attachments/assets/bb34977c-8d6c-4117-92b1-88e2e7e517a3)
Because appear the scrollbar. See https://github.com/dotnet/maui/blob/a407b5fbb0afe0266c1a01f864143c98b488599d/src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue18896.cs#L34

Reply:
@jsuarezruiz, thank you for the reference. I have made modifications based on it. Could you please review and share your feedback?